webplugin/plugin.js

```diff
@@ -72,6 +72,7 @@ function createKommunicateIframe() {
   kommunicateIframe.setAttribute("style", "overflow:hidden;"); // to fix scrollbars appearing before the chat widget loads on slow connections
   kommunicateIframe.setAttribute("scrolling", "no"); // to fix scrollbars appearing before the chat widget loads on slow connections
   kommunicateIframe.setAttribute("id", "kommunicate-widget-iframe");
+  kommunicateIframe.setAttribute("title", "Kommunicate live chat");

```

Comment:
@CaptainT33mo remove "kommunicate"

Reply:
@devashishmamgain Should I just keep it `Live chat`?

Reply:
yes we can keep "Live chat"